chainreg/chainregistry.go

```diff
@@ -485,11 +486,93 @@ func NewPartialChainControl(cfg *Config) (*PartialChainControl, func(), error) {
 
 		// The api we will use for our health check depends on the
 		// bitcoind version.
-		cmd, err := getBitcoindHealthCheckCmd(chainConn)
+		cmd, ver, err := getBitcoindHealthCheckCmd(chainConn)
 		if err != nil {
 			return nil, nil, err
 		}
 
+		// If the getzmqnotifications api is available (was added in
+		// version 0.17.0) we make sure lnd subscribes to the correct
+		// zmq events. We do this to avoid a situation in which we are
+		// not notified of new transactions or blocks.
+		if ver >= 170000 {
+
```

Comment:
nit: extra nl

Reply:
@bhandras Thank you for your review. Nit fixed.